printer/contexts.go

```diff
@@ -0,0 +1,72 @@
+package printer
+
+import (
+	"fmt"
+	"os"
+	"sort"
+
+	"github.com/olekukonko/tablewriter"
+	"github.com/qlik-oss/corectl/internal"
+)
+
+// PrintContext prints all information in a context
+func PrintContext(name string, context *internal.Context) {
+	if context == nil {
+		fmt.Println("No current context")
```

Comment:
Misleading name when you specify a named context to display

Reply:
Fixed